trulens_eval/trulens_eval/feedback/prompts.py

```diff
@@ -131,26 +131,22 @@
 
 please answer with this template:
 
-(Step 1)
 Important Points: <Outline the important points>
 
 Step 2 - Compare:
 (DO NOT SKIP THIS STEP!)
 
-For a SUMMARY How well does this summary address the above main points?
-
-please answer with this template:
-
-(Step 2)
-Supporting Evidence: <For each of the Important Points, explain if the SUMMARY does or does not mention it.>
-Score: <Give a score from 0 to 10 on if the SUMMARY addresses every single one of the main points. A score of 0 is no points were mentioned. A score of 5 is half the points were mentioned. a score of 10 is all points were mentioned.>
```

Comment:
Do we not need this rating rubric any more?

Reply:
Just using the standard COT template instead.